.github/workflows/ci.yml

```diff
@@ -173,6 +173,22 @@ jobs:
           name: nuget-package
           path: ./Bin/Release/ILGPU.*.*nupkg
 
+  release:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: [check-style, build-and-test]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download NuGet package artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: nuget-package
```

Comment:
could you please check if this should be `nuget-package` or `nuget-packages`? 💭 
per current `ci.yml` they are uploading [nuget-packages](https://github.com/m4rs-mt/ILGPU/blob/master/.github/workflows/ci.yml#L192) so you would receive here empty file? 🤔

Reply:
nice catch, it was updated meanwhile, will adapt, ty ⚡